src/main/java/org/apache/mesos/api/JettyApiServer.java

```diff
@@ -0,0 +1,55 @@
+package org.apache.mesos.api;
+
+import org.apache.mesos.scheduler.plan.api.PlanResource;
+import org.apache.mesos.scheduler.recovery.api.RecoveryResource;
+import org.apache.mesos.state.api.StateResource;
+import org.eclipse.jetty.server.Server;
+import org.glassfish.jersey.jetty.JettyHttpContainerFactory;
+import org.glassfish.jersey.server.ResourceConfig;
+
+import javax.ws.rs.core.UriBuilder;
+import java.net.URI;
+import java.util.Arrays;
+import java.util.Collection;
+
+/**
+ * A JettyApiServer takes a list of POJO JAX-RS Resources and serves them on the indicated port of 0.0.0.0.
+ */
+public class JettyApiServer {
```

Comment:
Think Protobuf helpers and py.test wrappers, and why we dislike them, and  why we ultimately removed Protobuf helpers ? And, want to get rid of py.text wrappers.

Config options like, but not limited to:

- Thread pool size
- Timeout
- Buffer size
- logging, etc

I don't want the users of this library to be limited from not being able to configure these operationally important things in there own code-base that they control and own.